### PR TITLE
Fixed #512: Added FAR-like archive files colors

### LINKED ISSUE #512: Highlight archive files with dark red color (like in FAR)

### PR DESCRIPTION
**Pull request by viktor-podzigun**

### CODE REVIEW

**Comment by corporateshark** [comment]
Weird. There is already a full-blown highlighting rules mechanism in place, why adding yet another ad hoc logic instead of using ``clNCFileHighlightingRule``?

Just add a function ``CreateDefaultHightlightingRules()`` and create a rule with ``m_Mask`` filled with archive extensions separated by commas and RGB colors in the appropriate fields.

File: src/panel.cpp [hunk 1]
```diff
@@ -891,6 +892,7 @@ void PanelWin::DrawItem( wal::GC& gc,  int n )
 
 	bool isDir = p && p->IsDir();
 	bool isExe = !isDir && p && p->IsExe();
+	bool isArch = !isDir && p && p->IsArch();
```
Comment (corporateshark) [inline src/panel.cpp]: Just use ``clNCFileHighlightingRule``.

**Comment by viktor-podzigun** [comment]
Indeed, its only about config, no code changes needed for this feature. Didn't know about it.
Will redo it by adding [HighlightingRules] config section. Thanks.

**Comment by corporateshark** [comment]
A section in the default file is ok, but it would be better to hard-reset it to default values. In case the file is corrupted.

**Comment by viktor-podzigun** [comment]
Not sure if I understood what you meant by "hard-reset it to default values". Could you be more specific, please.

**Comment by corporateshark** [comment]
For example, if the configuration file is broken or deleted. Users should be able to return to default values.

**Comment by viktor-podzigun** [comment]
Thats why we have default config, no?

**Comment by corporateshark** [comment]
Yes. However, there is no consistent way (except reinstalling) to restore everything.

**Comment by viktor-podzigun** [comment]
Well, I think this PR is only about adding colors config for archive files.
If you think I should do more here, just give me some concrete hints, and I will try to add it.

Otherwise, if you are talking about general config restoring, then lets open separate issue and discuss it there.

**Comment by corporateshark** [comment]
Sure, just add your new values to the default configuration file. It will solve this issue.

**Comment by viktor-podzigun** [comment]
I already add it to
```bash
install-files/share/wcm/config.default
```
of you meant another file?

**Comment by corporateshark** [comment]
Looks good.

**Pull request merged**